pokecli.py

```diff
@@ -69,23 +64,22 @@ def main():
 
     colorama.init()
 
-    config = init_config()
-    if not config:
+    config_dir = init_config()
+    if not config_dir:
         return
 
-    logger.log('[x] PokemonGO Bot v1.0', 'green')
-    logger.log('[x] Configuration initialized', 'yellow')
+    kernel.set_config_file(config_dir)
+    kernel.boot()
 
     try:
-        bot = PokemonGoBot(config)
+        bot = kernel.container.get('pokemongo_bot')
```

Comment:
Also seems like if I just `import pokemongo_bot` in order to fix the exception, no plugins work properly:

```
[2016-08-15 07:35:49] [Navigation] Walking towards PokeStop "Mariners Gate Central Park" (0.01km away, eta 24.00 seconds)
[2016-08-15 07:35:49] [Navigation] Arrived at PokeStop "Mariners Gate Central Park" (0.01km away)
[2016-08-15 07:35:51] [Navigation] Walking towards PokeStop "Twin Owls" (0.03km away, eta 11.00 seconds)
[2016-08-15 07:35:51] [Navigation] Arrived at PokeStop "Twin Owls" (0.03km away)
[2016-08-15 07:35:52] [Navigation] Walking towards PokeStop "Four Greek Gods" (0.07km away, eta 32.00 seconds)
[2016-08-15 07:37:04] [Navigation] Arrived at PokeStop "Four Greek Gods" (0.07km away)
[2016-08-15 07:37:05] [Navigation] Walking towards PokeStop "Summit Rock" (0.05km away, eta 1.25 minutes)
```